examples/scripts/cycling_ageing.py

```diff
@@ -66,14 +66,15 @@
         "Discharge capacity [A.h]",
         "Electrolyte potential [V]",
         "Electrolyte concentration [mol.m-3]",
-        "Negative total SEI thickness [m]",
-        "Negative electrode porosity",
         "X-averaged negative electrode porosity",
         "X-averaged negative electrode SEI interfacial current density [A.m-2]",
+        "X-averaged negative electrode lithium plating interfacial current density "
+        "[A.m-2]",
         "X-averaged negative total SEI thickness [m]",
-        [
+        "X-averaged dead lithium concentration [mol.m-3]"[
```

Comment:
I am guessing this is the fix for the pre-commit issues
```suggestion
        "X-averaged dead lithium concentration [mol.m-3]",
        [
```